ios/RNZoomUs.m

```diff
@@ -129,9 +125,14 @@ - (dispatch_queue_t)methodQueue
       ms.delegate = self;
 
       MobileRTCMeetingJoinParam * joinParam = [[MobileRTCMeetingJoinParam alloc]init];
-      joinParam.userName = displayName;
-      joinParam.meetingNumber = meetingNo;
-//       joinParam.password = pwd;
+      joinParam.userName = data[@"userName"];
+      joinParam.meetingNumber = data[@"meetingNumber"];
+      joinParam.password =  data[@"password"];
+      joinParam.participantID = data[@"participantID"];
+      joinParam.zak =  data[@"zak"];
+      joinParam.webinarToken =  data[@"webinarToken"];
+      joinParam.noAudio = data[@"noAudio"];
+      joinParam.noVideo = data[@"noVideo"];
```

Comment:
1. keep the order of code the same as on android - it will be easier to follow and check.
2. also I think it makes sense to comment on android things that are not available there but implemented on ios (and vice versa)

Reply:
1) I keep IOS side similar to zoom-ios-sdk examples, so it will be easier to maintain updates.
I don't remember about android side. But it works different. There is 2 specific params.
I think we will have time in future to update it. There are still many unsupported properties.

2) I commented these fields within ts file.